packages/chord-mark/tests/unit/renderer/components/renderLine.spec.js

```diff
@@ -80,3 +80,81 @@ describe.each([
 		});
 	});
 });
+
+describe.each([
+	[
+		'line only',
+		{
+			shouldOpenSection: false,
+			shouldClosePriorSection: false,
+			shouldCloseFinalSection: false,
+			sectionWrapperClasses: [],
+		},
+		'^<p class="cmLine">%l</p>$',
+	],
+
+	[
+		'opens section only',
+		{
+			shouldOpenSection: true,
+			shouldClosePriorSection: false,
+			shouldCloseFinalSection: false,
+			sectionWrapperClasses: ['cmSection','cmSection-intro'],
+		},
+		'^<div class="%c"><p class="cmLine">%l</p>$',
```

Comment:
same here:
`<div class="cmSection cmSection-intro"><p class="cmLine">myLine</p>`

I understand it might look a bit redundant but it's usually better to have "dumb" tests rather than "smart" tests with extra logic.

Reply:
Good feedback.